src/frontend/js/widgets/Dashboard/components/TeacherProfileDashboardSidebar/components/OrganizationThumb/_styles.scss

```diff
@@ -0,0 +1,18 @@
+.organization-thumb {
+  border-radius: 8px;
+  border: 1px solid r-theme-val(organization-thumbs, border-color);
+  overflow: hidden;
+  width: 73px;
+  height: 40px;
```

Comment:
Maybe something better to do ? Because currently if thumbnail are not flexible, I feel manage responsive will be a nightmare.

Reply:
They are quite small, i don't see the need to have them adapt they size.